public/index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Print a card for your WiFi login" />
+    <link rel="icon" href="../src/images/wifi.ico">
```

Comment:
The favicon isn't loading for me in Firefox or Chrome. It might need to be placed in the `public` directory?

<img width="646" alt="image" src="https://user-images.githubusercontent.com/4248167/125803974-613cd0ee-8b39-40c1-bebe-d28b796c7577.png">